package.json

```
@@ -30,7 +30,8 @@
     "test:unit:watch": "npm run test:unit -- --watch",
     "test:unit:ci": "npm run test:unit -- --ci --coverage",
     "test:percy": "if [ $PERCY_TOKEN ]; then percy-storybook; else echo 'Skipping test: PERCY_TOKEN is not defined'; fi",
-    "coverage": "cat ./coverage/lcov.info | coveralls -v"
+    "coverage": "cat ./coverage/lcov.info | coveralls -v",
+    "test:unit:coverage": "npm run test:unit --coverage && open coverage/lcov-report/index.html"
```

```suggestion
    "test:unit:coverage": "npm run test:unit -- --coverage && open coverage/lcov-report/index.html"
```

I think this change is needed

---

👍